src/distribusjonstyper.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Kodeverk/distribusjonstyper
+ * @module
+ */
+const distribusjonstyper = [
+  {
+    kode: 'VEDTAK',
+    term: 'VEDTAK',
```

Comment:
Term er stort sett noe som er leselig for bruker/saksbehandler. Dersom denne kun skal brukes i melosys-api, kunne den heller vært en enum i api?

Reply:
Ja, tenkte det samme egentlig. Kan være enig i at kodevern var unødvendig å ha i. Kan endre etterpå.